docs/GraFx-Studio/guides/connector-hub/index.md

```diff
@@ -0,0 +1,53 @@
+# Connector Hub
+
+## Intro
+
+The Connector Hub is the central repository where you can find connectors you can install into your environment.
+
+## Overview of installed connectors
+
+As an Environment Admin, navigate to:
+
+My Environments > [Your Environment] > Settings > Connectors
+
+![screenshot-full](ch01.png)
+
+Select the tab Connectors
+
+![screenshot-full](ch02.png)
+
+Overview of the installed connectors
+
+![screenshot-full](ch03.png)
+
+## Add a connector
+
+Navigate to the list of installed connectors (see above) and click "+ Add connector"
+
+A pop-up window will display.
+
+![screenshot-full](ch04.png)
+
+Choose "Connector Hub". A list of available connectors will show.
+
+![screenshot-full](ch07.png)
+
+Choose the connector you want to deplor or install in your environment.
+
+For detailed configuration parameters, continue on the relevant connector page in documentation.
+
+- [Acquia DAM](/GraFx-Studio/connectors/connect-acquia/)
+
+## Activate / De-activate
```

Comment:
I would rephrase to how we describe it in the UI: 'available'
And I would add more info about this:

- **Available**: the connector is available in the TD Workspace

**- Not available**:
- ‘not available’ is the [default] state

- if the connector was available in the past but made not available again:
- the connector remains functionally in the WRS & EU Interface ( = we DON’T break existing template)
- but you can’t add a new variable with the connector 
- you can update existing variables with the connector

Reply:
Added the info